runtime/rasdump/javadump.cpp

```diff
@@ -5291,7 +5292,29 @@ JavaCoreDumpWriter::writeCPUinfo(void)
 	_OutputStream.writeCharacters(
 			"2CITARGETCPU   Target CPUs: ");
 	_OutputStream.writeInteger(target, "%i\n");
+	
+	int BUFF_SIZE = 400;
+	char buff[BUFF_SIZE];
+	if (_VirtualMachine->jitConfig) {
+		_OutputStream.writeCharacters(
+				"2CIJITFEATURE  CPU features (JIT): ");
+		omrsysinfo_get_processor_feature_string(&_VirtualMachine->jitConfig->targetProcessor, buff, BUFF_SIZE);
+		_OutputStream.writeCharacters(buff);
```

Comment:
If the buffer isn't big enough and the string is cut off, it won't have a `\n` at the end.

Reply:
I think it would be better, more flexible, if omrsysinfo_get_processor_feature_string() didn't include the `\n` and that was handled here. Not every use of omrsysinfo_get_processor_feature_string() may want `\n` at the end. Ideally there would also be a return value to indicate if the buffer was big enough or not.